src/components/Excerpt.tsx

```diff
@@ -5,21 +5,48 @@ import { useCallback, useLayoutEffect, useRef, useState } from 'preact/hooks';
 
 import { observeElementSize } from '../utils/observe-element-size';
 
-type InlineControlsProps = {
+type InlineControlExcerptProps = {
+  /**
+   * The excerpt provides internal controls to expand and collapse
+   * the content.
+   */
+  inlineControl: true;
+
+  /**
+   * Text on inline control when clicking it will expand the content.
+   * Defaults to 'More'.
+   */
+  inlineControlExpandText?: string;
```

Comment:
This is fine. Just as a thought, an alternative API would have been to set the type of `inlineControl` to `true | false | { expandText?: string; collapseText?: string; ... }` where `true` means "enable inline controls with default options" and the last variant means "enable with custom options".